### PR TITLE
Close files after value is read

### PR DESCRIPTION
**Pull request by pakb**
Was generating random concurrency related errors. Reverting back to what was there before https://github.com/geoadmin/service-alti/pull/46 (keeping the close on delete, doesn't hurt if something went wrong during reading)

fixes https://github.com/geoadmin/service-alti/issues/65

### CODE REVIEW

File: alti/lib/raster/georaster.py [hunk 1]
```diff
@@ -98,9 +98,8 @@ def get_height_for_coordinate(self, x, y):
                 format_to_unpack = "<h"
             else:
                 format_to_unpack = "<l"
+        self.close_file()
         return unpack(format_to_unpack, self.file.read(self.data_size))[0]
```
Comment (loicgasser) [inline alti/lib/raster/georaster.py:102]: you might want to close the file after you read the height value.

```python
height = unpack(format_to_unpack, self.file.read(self.data_size))[0]
self.close_file()
return height
```

Comment (pakb) [reply alti/lib/raster/georaster.py:102]: yeah right... silly me... thanks for the feedback

File: alti/lib/raster/georaster.py [hunk 2]
```diff
@@ -98,9 +98,9 @@ def get_height_for_coordinate(self, x, y):
                 format_to_unpack = "<h"
             else:
                 format_to_unpack = "<l"
-        return unpack(format_to_unpack, self.file.read(self.data_size))[0]
-        # we let the file opened, so that we can read further point more efficiently
-        # (the file will be closed by a call to close_file() or by the __del__ function)
+        value = unpack(format_to_unpack, self.file.read(self.data_size))[0]
+        self.close_file()
```
Comment (boecklic) [inline alti/lib/raster/georaster.py:102]: Using pythons `with ...` context manager accessing resources handles closing automatically

Comment (procrastinatio) [reply alti/lib/raster/georaster.py:102]: You may also use `try..except..finally`

**Pull request merged**